docs/source/admin/traffic_router.rst

```diff
@@ -681,8 +681,67 @@ The ordering of certificates within the certificate bundle matters. It must be:
 
 To see the ordering of certificates you may have to manually split up your certificate chain and use :manpage:`openssl(1ssl)` on each individual certificate
 
-Suggested Way of Setting up an HTTPS Delivery Service
------------------------------------------------------
+Let's Encrypt
+-------------
+Let’s Encrypt is a free, automated :abbr:`CA (Certificate Authority)` using :abbr:`ACME (Automated Certificate Management Environment)` protocol. Let's Encrypt performs a domain validation before issuing or renewing a certificate. There are several options for domain validation but for this application the DNS challenge is used in order to receive wildcard certificates. Let's Encrypt sends a token to be used as a TXT record at ``_acme-challenge.domain.example.com`` and after verifying that the token is accessible there, will return the newly generated and signed certificate and key. The basic workflow implemented is:
+
+#. ``POST`` to Let's Encrypt and receive the DNS challenge token.
+#. Traffic Ops stores the DNS challenge in the Traffic Ops database.
+#. Traffic Router has a watcher set up to watch for changes in the Traffic Ops database table.
```

Comment:
I hate this wording where you keep saying 'database'. It sounds like TR is performing SQL queries on Postgres. I think its fine if you just say something like 'Traffic Ops stores the DNS challenge and then Traffic Router has a watcher which checks with Traffic Ops for any new challenges." I think you should fix all the references to 'database'.

Reply:
I tend to agree. The database is just an implementation detail of Traffic Ops, users/operators shouldn't need to even be aware of its existence.

Reply:
done